Samples/InpaintHTTP/WebHost.cs

```diff
@@ -0,0 +1,131 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Nancy;
+using Nancy.Hosting.Self;
+using Nancy.ErrorHandling;
+using System.Linq;
+using Zavolokas.GdiExtensions;
+using Zavolokas.ImageProcessing.Inpainting;
+using Zavolokas.Structures;
+using System.Threading.Tasks;
+using Zavolokas.ImageProcessing.PatchMatch;
+using System.Drawing;
+using System.IO;
+using System.Threading;
+//using Zavolokas.Utils.Processes;
+
+//TODO: Cleanup the mess above & unused References
+
+namespace InpaintHTTP
+{
+    class WebHost
+    {
+        public WebHost()
+        {
+            NancyHost host = new NancyHost(new Uri("http://localhost:8069"));
+            host.Start();
+            Console.WriteLine("Initialized and started WebHost");
+        }
+    }
+
+    public class MainMod : NancyModule
+    {
+        public MainMod()
+        {
+            Post("/api/inpaint", async x =>
+            {
+                Console.WriteLine("[" + DateTime.Now.ToString("HH:mm:ss.fffff") + "] Incomming request from " + this.Request.UserHostAddress);
+                if (this.Request.Files.Count() > 1)
+                {
+                    Bitmap BitmapImg;
+                    Bitmap BitmapMask;
+
+                    byte[] ByteImg = new byte[this.Request.Files.First().Value.Length];
+                    this.Request.Files.First().Value.Read(ByteImg, 0, (int)this.Request.Files.First().Value.Length);
+                    using (MemoryStream ms = new MemoryStream(ByteImg))
+                    {
+                        BitmapImg = new Bitmap(ms);
+                    }
+                    byte[] ByteMask = new byte[this.Request.Files.Last().Value.Length];
+                    this.Request.Files.Last().Value.Read(ByteMask, 0, (int)this.Request.Files.Last().Value.Length);
+                    using (MemoryStream ms = new MemoryStream(ByteMask))
+                    {
+                        BitmapMask = new Bitmap(ms);
+                    }
+
+                    var imageArgb = ConvertToArgbImage((Bitmap)BitmapImg);
```

Comment:
`BitmapImg` is already of type of `Bitmap` as defined at the top.
```suggestion
                    var imageArgb = ConvertToArgbImage(BitmapImg);
```